src/PrestaShopBundle/Resources/config/services/core/domain/country.yml

```diff
@@ -1,6 +1,11 @@
 services:
   _defaults:
-    public: true
```

Comment:
maybe you can keep public: false to tell explicitly which visibility is used by default. I think It's also a security in case of update of this default value.